domains/devrostudio.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "DevroStudio",
+    "email": "devrostudio2k25@gmail.com"
+  },
+  "records": {
+    "CNAME": "portafolio-e5e.pages.dev"
```

Comment:
This website appears to be for commercial use, which is against our terms of service. See pull request conditions. You can comment onto the end of this pull request if you think this is mistaken.